scripts/memory_logger.py

```diff
@@ -558,6 +541,11 @@ def __init__(self, arguments, machine_id):
                         }
                       }
 
+    if self._darwin() == True:
+      self.stack_trace = LLDB()
+    else:
+      self.stack_trace = GDB()
```

Comment:
This is an odd branch. You are tying the compiler type to the OS? What if we want to use Clang on Linux?